packages/storage/src/list.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Reference } from './reference';
+
+/**
+ * @fileoverview Documentation for ListOptions and ListResult format.
+ */
+export type ListOptions = {
```

Comment:
Can you check if the the tooling that we use for documentation generation will attach the comment above to `ListOptions`? I am not sure how `@fileoverview` works and if you need an empty line in between.

Reply:
I realized that I need to add those types to `toc.yaml` in order for `docgen` to work